mne/annotations.py

```diff
@@ -912,14 +918,22 @@ def events_from_annotations(raw, event_id="auto",
     event_sel, event_id_ = _select_annotations_based_on_description(
         annotations.description, event_id=event_id, regexp=regexp)
 
+    if return_durations:
+        fs = raw.info["sfreq"]
```

Comment:
please the common param names in mne. fs -> sfreq

Reply:
Done